app/workers/articles/dev_feeds_import_worker.rb

```diff
@@ -0,0 +1,17 @@
+# NOTE: [rhymes]
+# This script will soon be removed. We need it to collect monitoring data on
+# Datadog about the production behavior of the `Feeds::Import` class
+module Articles
+  class DevFeedsImportWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :medium_priority, retry: 10
+
+    def perform
+      return unless SiteConfig.community_name == "DEV"
+      return if Rails.cache.read("cancel_feeds_import").present?
```

Comment:
I may have missed it but where does this cache get written?

Reply:
It's just an escape plan @mstruve thought about. Basically if the worker restarts and we don't want it to, by writing that key manually, it will stop